src/main/java/org/reaktivity/nukleus/kafka/internal/memory/DefaultMemoryManager.java

```diff
@@ -127,7 +145,8 @@ public long acquire(
             }
         }
 
-        return ((nodeIndex + 1) & ~highestOneBit(nodeIndex + 1)) << blockSizeShift << nodeOrder;
+        long addressToShift = ((nodeIndex + 1) & ~highestOneBit(nodeIndex + 1));
+        return addressToShift << blockSizeShift << nodeOrder;
```

Comment:
Looks like this was a no-op refactor except to promote to `long`, but using `Long.highestOneBit(...)` instead of `Integer.highestOneBit(...)` makes the return value type `long` already.

Please verify that behavior remains unchanged when using `Long.highestOneBit(...)` instead and if so, then eliminate unnecessary diff.

Reply:
Verified and fixed